nwb_conversion_tools/utils/json_schema.py

```diff
@@ -312,7 +312,7 @@ def get_schema_for_NWBFile():
             "description": "Experimental protocol, if applicable. E.g., include IACUC protocol"
         },
         "related_publications": {
-            "type": "string",
+            "type": ["string", "array"],
```

Comment:
this is unrelated but a minor error that I found. Nwb specs have `related publications` as a str, list, tuple
https://pynwb.readthedocs.io/en/stable/pynwb.file.html#pynwb.file.NWBFile

Reply:
@Saksham20 Nice catch, can you work with Luiz on a separate PR for this?